README.md

```diff
@@ -6,5 +6,13 @@ This is a template for writing a thesis or dissertation using LaTeX at Oklahoma
 # Contributing
 Spot an error, bug, or want to contribute? Create a [branch](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) and submit a [pull request.](https://github.com/mitchute/OSULaTeXTheisTemplate/compare?expand=1)
 
+# Usage
+This template has been successfully utilized by many OSU graduate students. The graduate college does provide a template of their own. However, any template could be used. The information in the following table may be useful for a student who is considering whether to utilize the graduate colleges template or this one. The table includes a non-exhaustive description of students who have utilized this template.

```

Comment:
> The graduate college does provide a template of their own.

They have a LaTeX template of their own? They had taken that down when I was there.

Reply:
In light of recent events (most likely your @mitchute experiences with them), the grad college did pay a couple of guys to come in and edit the existing template. 
```
Original author: Brian O'Dell (1999), revised by Ryan T. Ratliff (2005).
Current version by Paul Fili, Edward Richmond (2020).
```
The grad colleges template does not have the organization of this template, nor does it have hyper-referencing for figures and the TOC/LOF/LOT. I believe this template is more user friendly.